frontend/package.json

```diff
@@ -17,6 +17,7 @@
     "nock": "^13.2.4",
     "react": "^17.0.1",
     "react-bootstrap": "^2.0.0-alpha.0",
+    "react-datepicker": "^7.5.0",
```

Comment:
Why are we adding this as a dependency when I don't see it used anywhere in the code?

I'm not opposed to adding this, but if you add it as a dependency, you need to actually use it somewhere in the PR.  So either use it, or lose it.

Reply:
Oh, and if you do decide to lose it, make sure that package-lock.json gets restored as well: 

See: https://ucsb-cs156.github.io/topics/pull_requests/package_lock_json.html

Reply:
I have removed react-datepicker from both package.json and package-lock.json.